### ISSUE TITLE
safari global.Symbol TypeError

### ISSUE THREAD

**Issue by felixroos** [medikoo/es6-symbol#31]
Since version ```3.1.2```, safari throws a TypeError when using Symbol:

```js
require("es6-symbol/implement");
const symbol = Symbol.for('xyz');
```

the following error is thrown in safari:

```
TypeError: undefined is not an object (evaluating 'global.Symbol')
```

when commenting out the require, it works. fixing to version 3.1.1 also works.

the release notes of 3.1.2 say: "Access Symbol from a global object. Makes implementation more bulletproof [...]". This seems like a bug to me

**Comment by medikoo** [comment]
@felixroos great thanks for report.

Can you share what tool you use to bundle `es6-symbol` for a Safari?

**Comment by medikoo** [comment]
If you can outline an exact steps to create a minimal test case script, that crashes as you say in Safari. That'll be valuable.

There's really no bug in the code, as `global` is reliably provided via `es5-ext`. I believe there's something not right going on in bundling process

**Comment by medikoo** [comment]
@felixroos great thanks. Indeed I've reproduced that, and it appears that due to Safari quirk, resolution of global didn't work properly in that engine.

It's fixed now and published with v3.1.3
